graphql/handler/transport/websocket.go

```diff
@@ -179,8 +179,10 @@ func (c *wsConnection) init() bool {
 			}
 		}
 
+		var initAckPayload *InitPayload = nil
```

Comment:
won't `new` do the same here?

Reply:
New actually doesn't do what I want here. I want a nil pointer so I can figure out if we actually have any payload or not.